pkg/asset/installconfig/clusterid.go

```diff
@@ -6,24 +6,25 @@ import (
 	"github.com/openshift/installer/pkg/asset"
 )
 
-type clusterID struct {
+// ClusterID is the unique ID of the cluster, immutable during the cluster's life
+type ClusterID struct {
```

Comment:
why not just `type ClusterID string` ?

Reply:
Just for uniformity of initialization, [here](https://github.com/openshift/installer/pull/783/files#diff-36a138ec46315c3afef6bca683543ee8R122) for example by replacing installconfig.ClusterID{} with new(installconfig.ClusterID)
Also looks kludgy to dereference and assign 'self' in a class function [here](https://github.com/openshift/installer/pull/783/files#diff-a394ea3cd524aeb0c4598833f69f5ed8R23) when it appears like `*a = uuid.New()`

The sizeof() should also not be different, but drop one hint and I can change it, I am fine either way.

Reply:
It makes sense to me to have `ClusterID` be a `struct`. The type is not the cluster ID itself: The type is the asset that generates the cluster ID.

Reply:
There is precedent for this in all the other assets. For comparison, `installconfig.baseDomain` *could* be a `string` instead of a `struct`, too.